core/storage/statistics/gae_models.py

```diff
@@ -346,7 +430,16 @@ def get_new_event_entity_id(cls, exp_id, user_id):
 
     @classmethod
     def create(cls, exp_id, user_id, rating, old_rating):
-        """Creates a new rate exploration event."""
+        """Creates a new rate exploration event and then writes it to the
+        datastore.
+
+        Args:
+            exp_id: str. ID of the exploration currently being played.
+            user_id: str. ID of the user.
+            rating: int. Value of rating assigned to exploration.
+            old_rating: int. Will be None if the user rates an exploration
```

Comment:
If it can be None. write type as `int or None.`

Reply:
I corrected this one.